engine/src/mblandroiddc.cpp

```diff
@@ -2229,6 +2229,16 @@ JNIEXPORT void JNICALL Java_com_runrev_android_Engine_doLaunchFromUrl(JNIEnv *en
     if (MCJavaStringToStringRef(env, url, &t_url_str))
         MCNotificationPostUrlWakeUp(*t_url_str);
 }
+
+extern "C" JNIEXPORT void JNICALL Java_com_runrev_android_Engine_doLaunchDataChanged(JNIEnv *env, jobject object, jobject launch_data) __attribute__((visibility("default")));
+JNIEXPORT void JNICALL Java_com_runrev_android_Engine_doLaunchDataChanged(JNIEnv *env, jobject object, jobject launch_data)
+{
+	
+	MCAutoArrayRef t_array;
+	if (MCJavaMapToArrayRef(env, launch_data, &t_array))
+		/* UNCHECKED */ MCNotificationPostLaunchDataChanged(*t_array);
```

Comment:
Do we need to pass a parameter to the notification? After all the new launch data is available via the standard function.